homeassistant/components/lutron/switch.py

```diff
@@ -50,3 +61,48 @@ def update(self):
         """Call when forcing a refresh of the device."""
         if self._prev_state is None:
             self._prev_state = self._lutron_device.level > 0
+
+
+class LutronLed(LutronDevice, SwitchDevice):
+    """Representation of a Lutron Keypad LED."""
+
+    def __init__(self, area_name, keypad_name, scene_device, led_device, controller):
+        """Initialize the switch."""
+        self._prev_state = None
+        self._keypad_name = keypad_name
+        self._scene_name = scene_device.name
+        super().__init__(area_name, led_device, controller)
+
+    def turn_on(self, **kwargs):
+        """Turn the LED on."""
+        self._lutron_device.state = 1
+
+    def turn_off(self, **kwargs):
+        """Turn the LED off."""
+        self._lutron_device.state = 0
+
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes."""
+        attr = {}
+        attr["keypad"] = self._keypad_name
+        attr["scene"] = self._scene_name
+        attr["led"] = self._lutron_device.name
+        return attr
+
+    @property
+    def is_on(self):
+        """Return true if device is on."""
+        return self._lutron_device.last_state
+
+    @property
+    def name(self):
+        """Return the name of the LED."""
+        return "{} {}: {} LED".format(
+            self._area_name, self._keypad_name, self._scene_name
+        )
+
+    def update(self):
+        """Call when forcing a refresh of the device."""
+        if self._prev_state is None:
+            self._prev_state = self.state
```

Comment:
this is why you don't sync at boot. This should be self._lutron_device.state

Reply:
@alistairg can you give this a look? Once you push another commit, I can test again

Reply:
Done - sorry this took a while. Work is a little nuts at the moment ;)

Changes pushed.